packages/be/crons/application-stats.js

```diff
@@ -0,0 +1,225 @@
+/**
+ *
+ * ⏱️️ [Cron | every 1 hour] ApplicationStats
+ *
+ */
+
+// ///////////////////////////////////////////////////// Imports + general setup
+// -----------------------------------------------------------------------------
+const ModuleAlias = require('module-alias')
+const Path = require('path')
+const Fs = require('fs-extra')
+const Axios = require('axios')
+
+require('dotenv').config({ path: Path.resolve(__dirname, '../.env') })
+
+const MC = require('../config')
+
+// ///////////////////////////////////////////////////////////////////// Aliases
+ModuleAlias.addAliases({
+  '@Root': MC.packageRoot,
+  '@Static': `${MC.packageRoot}/static`,
+  '@Public': `${MC.packageRoot}/public`,
+  '@Cache': `${MC.packageRoot}/cache`,
+  '@Modules': `${MC.packageRoot}/modules`
+})
+
+try {
+  const modulesRoot = `${MC.packageRoot}/modules`
+  const items = Fs.readdirSync(modulesRoot)
+  items.forEach((name) => {
+    const path = `${modulesRoot}/${name}`
+    if (Fs.statSync(path).isDirectory()) {
+      const moduleName = (name[0].toUpperCase() + name.substring(1)).replace(/-./g, x => x[1].toUpperCase())
+      ModuleAlias.addAlias(`@Module_${moduleName}`, path)
+    }
+  })
+} catch (e) {
+  console.log(e)
+}
+
+// ///////////////////////////////////////////////////////////////////// Modules
+const { SecondsToHms } = require('@Module_Utilities')
+
+// /////////////////////////////////////////////////////////////////// Functions
+// -----------------------------------------------------------------------------
+// ////////////////////////////////////////////////////////////// getTotalIssues
+const getTotalIssues = async (repo, dataProgramsToken) => {
+  // console.log('🦞 > 🦀 getting total issues')
+  try {
+    const options = {
+      headers: { Accept: 'application/vnd.github+json', 'X-GitHub-Api-Version': '2022-11-28', Authorization: `Bearer ${dataProgramsToken}` }
+    }
+    const response = await Axios.get(`https://api.github.com/search/issues?q=repo:${repo}%20is:issue`, options)
+    return response.data.total_count
+  } catch (e) {
+    console.log('================================== [Function: getTotalIssues]')
+    console.log(e)
+    throw e
+  }
+}
```

Comment:
Let's get rid of this `getTotalIssues` call and make the `getApplications` function recursion be based off the length of the returned array of issues